### DISCUSSION TITLE
add a option to reset data from ticket to restart to ticket 1

### DISCUSSION THREAD

**Discussion by sadoj-kuro** [discord-tickets/bot#348] [General]
add a option to reset data from ticket to restart to ticket 1

**Comment by eartharoid** [comment]
Resetting the database (by deleting and re-creating) has this effect, although it resets the settings as well, and for all of the Discord servers the bot is in.

There's a [reset button](https://github.com/discord-tickets/bot/issues/174) that will reset just a single Discord server in v4.

If you want to *only* reset the counter and not the settings, there is a suggestion for that at https://dsctickets.feedbacky.net/i/editable-counters.4256 that you can upvote (although it is unlikely to be added soon).
